src/components/Square.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 const Square = (props) => {
   return (
-    <div className="square"></div>
+    <div className="square">Anyt??</div>
```

Comment:
Did you want the inner HTML here to be hardcoded to "Anyt??" ?

Reply:
No. Debugging...